mrp_subcontracting_partner_management/readme/CONFIGURE.rst

```diff
@@ -1 +1 @@
-* No configuration is required
+To see newly created rules, go to **Settings** > **Inventory** and activate "Multi step routes".
```

Comment:
It is also possible to allow a user to check inventory locations in Sales & Purchases tab of Vendor without activating dev mode. To do so, enable technical extra right "Display Inventory section on Vendor" in user.

Reply:
Added